tools/CreateMappings_rules.json

```diff
@@ -148,6 +148,7 @@
     { "regex": "Advisor", "alias": "Advisor" },
     { "regex": "MixedReality", "group": "Mixed Reality", "alias": "Mixed Reality" },
     { "regex": "SpatialAnchors", "group": "Mixed Reality", "alias": "Mixed Reality" },
-    { "regex": "Blueprint", "alias": "Blueprint" },    
+    { "regex": "Blueprint", "alias": "Blueprint" },
+    { "regex": "NetAppFiles", "alias": "NetApp Files" },
     { "regex": "DeploymentManager", "alias": "Deployment Manager" }
 ]
```

Comment:
You will need to publish your SDK and remove the LocalFeed packages before this PR can be merged.

Reply:
The SDK is in the Azure repo. The localfeed SDK was produced with a sign job which was specially requested by Cormac. I'm confused as to the process here. Please clarify.